apps/stats-mainnet/src/components/images/light-mode-toggle.tsx

```diff
@@ -0,0 +1,10 @@
+export const LightModeToggle = () => {
```

Comment:
chore (blocking): Use UI toolkit switcher that now exists

Reply:
This isn't currently compatible with the way I've set up the dark mode, will deal with this as part of #113